deptofdefense/move.mil
# Entitlement search with radio buttons

Fixes #116.

## Checklist

I have…

- [x] Attached screenshots showing the relevant pages both before and after my changes.

## Summary of Changes

According to the USWDS, dropdowns should be used "sparingly — only when a user needs to choose from about seven to 15 possible options and you have limited space to display the options... If the list of options is very short[,] use radio buttons instead."

This changeset therefore replaces the second and third dropdowns on the Entitlements page with radio buttons. For an extra flourish, the CONUS/OCONUS radio buttons are images instead of boring circles + text.

Also, this changeset hides the Spouse Pro-Gear row from the results table if the user selected "No" for dependents.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `entitlements-with-radios`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. run `bin/rails server`, and
1. load up [http://localhost:3000](http://localhost:3000) in the Web browser of your choice.

## Screenshots

<details><summary>Desktop</summary>

![fireshot capture 69 - entitlements move mil - http___localhost_3000_entitlements](https://user-images.githubusercontent.com/26554826/35158709-f64f6b6c-fd05-11e7-9660-21791dcffb3f.png)

</details>

<details><summary>Mobile</summary>

![fireshot capture 71 - entitlements move mil - http___localhost_3000_entitlements](https://user-images.githubusercontent.com/26554826/35158716-fd3d3e40-fd05-11e7-9d05-c1999907c511.png)

</details>
